mne_bids_pipeline/steps/preprocessing/_03_maxfilter.py

```diff
@@ -468,8 +468,9 @@ def run_maxwell_filter(
 
         bids_path_ref_sss = in_files.pop("raw_ref_run_sss")
         raw_exp = mne.io.read_raw_fif(bids_path_ref_sss)
-        rank_exp = mne.compute_rank(raw_exp, rank="info")["meg"]
-        rank_noise = mne.compute_rank(raw_sss, rank="info")["meg"]
+        type_sel = set(cfg.ch_types).intersection({"mag", "meg"}).pop()
```

Comment:
I think this is more correct / general
```suggestion
        if "grad" in raw_exp:
            if "mag" in raw_exp:
                type_sel = "meg"
            else:
                type_sel = "grad"
        else:
            type_sel = "mag"
```